### PR TITLE
Dont reboot after restart

### PR DESCRIPTION
**Pull request by alimhassam**

### CODE REVIEW

File: bin/oref0-radio-reboot.sh [hunk 1]
```diff
@@ -15,19 +15,24 @@ EOF
...
-        echo >> $logfile
-    fi
-else
-    if [ -e /run/nologin ]; then
-        echo "No more radio errors; canceling reboot" | tee -a $logfile
-        shutdown -c | tee -a $logfile
-    fi
+SYSTEM_UP=$(date --date="$(uptime -s)" +%s)
+# Start checking only when the system has been up for 15 minutes.
+START_CHECKING=$(date -d"30 minutes ago" +%s)
```
Comment (scottleibrand) [inline bin/oref0-radio-reboot.sh]: 30 or 15?

Comment (alimhassam) [reply bin/oref0-radio-reboot.sh]: let me know, what's a reasonable number here, I thought the general rule was that the loop might take up to 20 min to start working so I decided to set it to 30 min to give it some time.

Comment (scottleibrand) [reply bin/oref0-radio-reboot.sh]: I don't really care, as long as the comment matches.

Comment (alimhassam) [reply bin/oref0-radio-reboot.sh]: Ha, oops. I'll take out the number from the comment.

Comment (alimhassam) [reply bin/oref0-radio-reboot.sh]: fixed comment.

**Pull request merged**